lib/orbit-common/jsonapi-source.js

```diff
@@ -585,7 +585,7 @@ var JSONAPISource = Source.extend({
       pathToVerify = operation.path;
     }
 
-    if (this.retrieve(pathToVerify)) {
+    if (this.retrieve(pathToVerify) !== null) {
```

Comment:
I'm not completely sure if comparing to `null` is correct here. `null` can also be valid value for existing data, but `retrieve()` returns `null` also if the path doesn't exist.